src/NServiceBus.Testing.Fakes/TestablePipelineContext.cs

```diff
@@ -111,5 +112,8 @@ static TimeoutMessage<object> GetTimeoutMessage(object message, SendOptions opti
 
         ConcurrentQueue<SentMessage<object>> sentMessages = new ConcurrentQueue<SentMessage<object>>();
         ConcurrentQueue<TimeoutMessage<object>> timeoutMessages = new ConcurrentQueue<TimeoutMessage<object>>();
+
+        // TODO: Figure out what to do with token in testing scenarios
+        public CancellationToken CancellationToken => CancellationToken.None;
```

Comment:
I guess the users might want to control the token when they are testing their handler and this achieves that right?